### PR TITLE
Fixed inaccurate Chinese words

### PR DESCRIPTION
**Pull request by fawdlstty**
**Describe the pull request**

- #### What does your PR fix?  
Fixed inaccurate Chinese words

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
 No

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  not added/updated port

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Review by JackBoosY — CHANGES_REQUESTED** [review]
能顺便根据英文文档更新一下其他内容吗？

**Comment by fawdlstty** [comment]
Ok, I will check the other documents later and then report back here.

File: CONTRIBUTING_zh.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# 贡献准则
+
+Vcpkg 是一个社区驱动的努力，旨在构建一个多产的、健壮的本地库生态系统 - 你的贡献价值不可估量！
```
Comment (JackBoosY) [inline CONTRIBUTING_zh.md:3]: "社区驱动的努力"?

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: CONTRIBUTING_zh.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+最简单的方法是通过 `vcpkg.exe` 或 [GitHub](https://github.com/Microsoft/vcpkg) 报告已有的包的问题。 当报告 `vcpkg.exe` 的问题时，确保清楚地说明:
+- 机器设置: “我用的是Windows 10周年更新。 我的机器正位于fr-fr区域。 我成功地运行了'install boost'。”
+- 复现步骤: “运行 'vcpkg list'”
+- 预期结果: “我预期看到 'boost:x86-windows'”
+- 实际结果: “没有输出” 或 “我得到一个崩溃对话框”
+
+当报告包的问题时，一定要清楚地说明:
+- 机器设置 (上述)
+- 你正在构建什么版本例如: “opencv 3.1.0”
```
Comment (Cheney-W) [inline CONTRIBUTING_zh.md:15]:
```suggestion
- 您正在构建什么包以及它的版本，例如: “opencv 3.1.0”
```

File: CONTRIBUTING_zh.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+- 构建过程中的任何相关错误日志
+
+## 贡献 (PR)
+
+我们很乐意接受关于修复、特性、新包和更新现有包的拉取请求。 为了避免浪费您的时间，我们强烈建议您公开讨论您正在考虑的PR是否可以接受。 对于特性和新包来说更是如此。
```
Comment (Cheney-W) [inline CONTRIBUTING_zh.md:20]:
```suggestion
我们很乐意接受关于修复、特性、新包和更新现有包的拉取请求。 为了避免浪费您的时间，我们强烈建议您提交一个问题来讨论您想要制作的PR是否能被接受。 对于特性和新包来说也是如此。
```

File: CONTRIBUTING_zh.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+### 新包贡献准则
+
+我们很高兴您有兴趣提交一个新的包! 这里有一些指导方针来帮助你编写一个优秀的作品集:
```
Comment (Cheney-W) [inline CONTRIBUTING_zh.md:24]:
```suggestion
我们很高兴您有兴趣来提交一个新的包! 这里有一些指导方针来帮助您编写一个优秀的端口文件:
```

File: CONTRIBUTING_zh.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+- 避免功能补丁。 当没有其他方法时，补丁应该被视为实现兼容性的最后手段。
+- 当无法避免补丁时，请不要修改默认行为。 一个补丁的理想生命周期是与上游合并，不再被需要。 在决定如何修补某些内容时，请记住这一目标。
+- 相比原始的 `execute_command` 调用，尽量改为通过 `vcpkg_xyz` 函数实现。这使得在添加新特性(如自定义编译器标志或生成器)时更容易进行长期维护。
+
+## 法律声明
+
+在你的pull请求被接受之前，你需要完成一个Contributor许可协议 (CLA)。 本协议证明您允许我们使用您提交的源代码，并且本作品是在合适的许可下提交的，我们可以使用它。
```
Comment (Cheney-W) [inline CONTRIBUTING_zh.md:31]:
```suggestion
在您的拉取请求被接受之前，您需要完成一个贡献者许可协议 (CLA)。 本协议证明您允许我们使用您提交的源代码，并且本作品是在合适的许可下提交的，我们可以使用它。
```

File: CONTRIBUTING_zh.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
+# 贡献准则
+
+Vcpkg 是一个尝试由社区驱动，旨在构建一个多产的、健壮的本地库生态系统 - 你的贡献价值不可估量！
```
Comment (Cheney-W) [inline CONTRIBUTING_zh.md:3]:
```suggestion
Vcpkg 是一个尝试由社区驱动，旨在构建一个多产的、健壮的本地库生态系统 - 您的贡献价值不可估量！
```

**Review by Cheney-W — CHANGES_REQUESTED** [review]

**Review by FrankXie05 — APPROVED** [review]

**Review by Cheney-W — APPROVED** [review]

**Review by JackBoosY — APPROVED** [review]

**Review by FrankXie05 — APPROVED** [review]

**Comment by JackBoosY** [comment]
Thanks for your contribution!

**Review by LilyWangLL — APPROVED** [review]

**Review by JonLiu1993 — APPROVED** [review]

**Review by Adela0814 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks everyone for the reviews :)

**Pull request merged**